bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/TpeContext.scala

```diff
@@ -77,7 +77,8 @@ final class TargetContext(obs: Option[ISPObservation]) extends TpeSubContext[ISP
 
   def envOrDefault: TargetEnvironment = env.getOrElse(TargetEnvironment.create(baseOrDefault))
 
-  def base: Option[SPTarget] = env.map(_.getBase)
+  // TODO: deal with multi-target asterisms
+  def base: Option[SPTarget] = env.map(_.getAsterism.allSpTargets.head)
```

Comment:
This would be another good place to mark with a "TODO ASTERISM" or some distinct label that we can easily search for later.